### PR TITLE
invalid duration updates

### PR DESCRIPTION
**Pull request by SFJohnson24**
this PR addresses the issue from @ASL-rmarshall 
- negative argument has been converted to a bool input
- the schema has been updated
- the operator.md reference has been updated to reflect this change
a seperate PR will be made to address the rule editor .md file

to test, use DDF60

### CODE REVIEW

File: resources/schema/Operator.json [hunk 1]
```diff
@@ -304,6 +304,9 @@
       "properties": {
         "operator": {
           "const": "invalid_duration"
+        },
+        "negative": {
```
Comment (ASL-rmarshall) [inline resources/schema/Operator.json:308]: I think `type` specifications for operator parameters are usually defined in the `properties` section, not within the `anyOf` section that specifies the allowable operations.  Within the `properties` section, the definitions are usually ordered alphabetically, so I'd recommend moving the `type` specification for `negative` to be between the specifications for `name` and `operator` (i.e., somewhere near line 906).

Comment (SFJohnson24) [reply resources/schema/Operator.json]: Thank you for pointing this out @ASL-rmarshall  I made the changes to the schema and pushed them to remote branch

Comment (ASL-rmarshall) [reply resources/schema/Operator.json]: The change looks good.

**Review by RamilCDISC — APPROVED** [review]
The PR has updated the code with all required changes in the connected issue AC. The problem of internal server error is solved . All related readme files are also updated. The changes integrate well with the system. DDF00060 has been used to validate the proper implementation

**Pull request merged**